lte/gateway/c/oai/lib/n11/SmfServiceClient.cpp

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright 2020 The Magma Authors.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "SmfServiceClient.h"
+#include "ServiceRegistrySingleton.h"
+#include <google/protobuf/util/time_util.h>
+
+using grpc::Status;
+
+namespace {
+
+void SetAmfSessionContextRpcCallback(
+    grpc::Status status, magma::lte::SmContextVoid response) {}
+
```

Comment:
Please handle negative return case from the rpc call. Put an error log message.

Reply:
e.g., check the status, if not okay then print out the error in error log. check the response message itself if it has any error codes, etc.

Reply:
Addressed